packages/shared/package.json

```diff
@@ -14,40 +14,30 @@
     "test:unit": "echo \"No tests yet\"",
     "fix:staged-files": "lint-staged --allow-empty",
     "fix:all-files": "eslint . --ext .ts,.tsx,.js,.jsx --fix",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
     "pre-commit": "echo 'No pre-commits defined in: shared'",
     "start": "tsc --build --clean && tsc --build && tsc --build tsconfig.json"
   },
   "devDependencies": {
     "@babel/core": "^7.22.9",
-    "@storybook/addon-actions": "^6.5.16",
-    "@storybook/addon-essentials": "^6.5.16",
-    "@storybook/addon-interactions": "^6.5.16",
-    "@storybook/addon-links": "^6.5.16",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/react": "^6.5.16",
-    "@storybook/testing-library": "^0.0.13",
-    "@types/node": "^20.4.2",
-    "@types/react": "^18.2.15",
+    "@types/node": "^20.4.5",
+    "@types/react": "^18.2.18",
     "babel-loader": "^9.1.3",
-    "eslint-config-prettier": "^8.8.0",
+    "eslint-config-prettier": "^8.9.0",
     "prettier": "^3.0.0"
   },
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@lifi/wallet-management": "^2.2.1",
-    "@mui/icons-material": "^5.14.0",
-    "@mui/material": "^5.14.0",
-    "i18next": "^22.5.0",
+    "@lifi/wallet-management": "^2.2.4",
+    "@mui/icons-material": "^5.14.3",
+    "@mui/material": "^5.14.3",
+    "i18next": "^22.5.1",
     "js-cookie": "^3.0.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-i18next": "^12.3.1",
     "typescript": "^5.1.6",
     "use-media": "^1.4.0",
```

Comment:
I guess we can remove `js-cookie` and `use-media`

Reply:
Good find, removed!